### ISSUE TITLE
🆕 Software Suggestion | ElementaryOS

### ISSUE THREAD

**Issue by LongJohn-Silver** [privacytools/privacytools.io#1967]
## Basic Information

**Name:** ElementaryOS
**Category:** Operative System
**URL:** https://elementary.io/

## Description
I know that the listing of this OS has been discussed to certain extent on #1130 and #1156, finding some rejection from some PT moderators, and some approval by others, but there is no current issue addressing specifically whether or not this is a good idea privacy wise.

## Why I am making the suggestion
I am making this suggestion after a period of about six months in which I switched from Windows 10 to Debian 10 in an effort to start using a more privacy respecting and libre operative system. I didn't know *anything* about how Linux OSes work, and my path was to figure it out while I was using it and to say that my experience wasn't good is to be kind. Of what is listed on your web site, everything requires quite a bit of understaning of command line use and different methods of installation on Linux, that without keeping in mind (even though this wasn't a thing I that I put above privacy) that the aesthetic part of them is not so welcoming as closed source OSes. 

I had to reinstall Debian three times because I fucked it up again and again creating an unstable experience which didn't work. This isn't an attack on Debian, I really loved certain parts of it and I'm sure it's a really awesome distribution, but it's not so user friendly for noobies.

ElementaryOS comes with an application store that has quite a lot more applications than Debian does, plus things like Firefox are not ESR (which is something horrible to use IMO), plus it comes with support for Flathub out of the box. You go to Flathub's website, click install on an application, a new window appears telling you a warning, you click yes, it installs it and after that all the applications that are on Flathub's website are on your application store, which is way more user friendly and it covers a lot of what most regular people need. That, and also their design is really welcoming and modern, in contrast with Trisquel which seems a bit outdated to say the least.

## My connection with the software

None.

- [x] I will keep the issue up-to-date if something I have said changes or I remember a connection with the software.

**Comment by dngray** [comment]
I think the direction we're looking at is https://github.com/privacytools/privacytools.io/issues/1842

As for ElementaryOS I do have a few problems with recommending that:

- [Labeling users "cheaters"](https://liliputing.com/2015/02/elementary-os-devs-say-youre-cheating-dont-pay-free-software.html) who don't donate or just want to try out your OS doesn't sit right with me. 
- I'm also not keen on them [taking large cut out of donations](https://developer.elementary.io/). to software on their "app store". I think 70/30 is a bit high to be honest. Google and Apple might get away with that for proprietary software, it's a commonly known fact that less people pay for free software. That doesn't really seem fair to the individual developers.

**Comment by LongJohn-Silver** [comment]
>I think the direction we're looking at is #1842

I don't know if you mean that you want to first improve that section, or if you made a mistake a gave me a wrong issue, but it's unrelated to OSes.

>Labeling users "cheaters" who don't donate or just want to try out your OS doesn't sit right with me.

I am the first one to say that libre software should remain free, because otherwise you lose the open access part of it which helps people that has low incomes or live in countries that are not so well positioned economically to use software which they couldn't in any other way. But at the same time I understand the position of a developer who receiving some money could help a lot with how your project evolves, we are all proletarians/cognitarians trying to survive capitalism after all. Nevertheless, on the article you showed me, the author mentions that the EOS devs have rephrased that blogs post, and if you try to access it now, it's gone, so I thinking that maybe they just worded their idea wrongly, and maybe they even changed their minds regarding to that, and that was *five* years ago, I think we could forgive them something like that.
I really like the idea that they give developers a choice to earn some through the application store, to be honest, and that they incentivize people to buy them.

>I'm also not keen on them taking large cut out of donations. to software on their "app store". I think 70/30 is a bit high to be honest. Google and Apple might get away with that for proprietary software, it's a commonly known fact that less people pay for free software. That doesn't really seem fair to the individual developers.

If you follow the link that you provide me, right under the Stripe section there's another link "Learn more about monetizing your app", and if you follow that one there's this text: 

>Platform Fees
>****
>
>Each payment is split between you and elementary 70/30, with a minimum payment to elementary of $0.50 to cover distribution and payment processing. For example: with a $10 payment, $3 is paid to elementary and $7 is paid to you. For a $1 payment, $0.50 is paid to elementary to cover distribution and payment processing while $0.50 is paid to you.
>
>There are no enrollment fees or subscription costs to publish your app in AppCenter.

So, basically, it's the other way around, devs get 70%, and EOS gets 30%, which seems pretty fair to me.

**Comment by danarel** [comment]
Elementary is becoming a popular OS but they are not focused on user privacy. While they may not be taking steps to be anti-private, the owners of the OS don't care about privacy at all. They regularly post about their new Google gadgets, etc and don't seem to keep privacy top of mind.

I don't think EOS has any place on this site mainly because we don't really list forks of Debian unless they are created specifically for the privacy community. Otherwise, we'd be much more keen to list Ubuntu over EOS.

Also, to the point about the 70/30 split, by default EOS also turns off the ability to add PPAs so that you're forced to use their app store in an effort to drive revenue. I don't like that.

So the inclusion of EOS is a huge no from me.

**Comment by LongJohn-Silver** [comment]
Elementary is becoming a popular OS but they are not focused on user privacy. While they may not be taking steps to be anti-private, the owners of the OS don't care about privacy at all. They regularly post about their new Google gadgets, etc and don't seem to keep privacy top of mind.

To be honest I don't know exactly what kind of gadgets the the owner uses, but recently they switched from Google Analytics to Plausible Analytics for their website, so that means that there's some acknowledgement of privacy in their being.
https://old.reddit.com/r/elementaryos/comments/hebmax/elementaryos_switches_from_google_analytics_to/
Also, there's this post that talks about eOS regarding privacy and how they do car about, even though if it's not _**the most**_ private Linux distro, also, on that post they stated that they moved from Medium to a blog on their website because of privacy concerns.
https://medium.com/elementaryos/privacy-and-elementary-os-95f9f2debe98

>Otherwise, we'd be much more keen to list Ubuntu over EOS.

Ubuntu has snaps on their app store and this is not good for security a lot of the times, since they are possibly outdated and not isolated, in contrast to eOS which uses Flatpaks, that, without mentioning other serious concerns regarding privacy on Ubuntu.
https://www.eff.org/deeplinks/2012/10/privacy-ubuntu-1210-amazon-ads-and-data-leaks

>by default EOS also turns off the ability to add PPAs so that you're forced to use their app store in an effort to drive revenue. I don't like that.

This is not so to force you to use the app store, eOS aims to be a distribution that is designed for new linux users. The default copy/paste on the terminal is ctrl+c/ctrl+v (instead of shift+ctrl+c/v), when you run a command with sudo privileges it gives you a warning that it is asking for such, and I assume that the ppa thing is related to that, so that you read a bit and understand what you are doing before playing around, because you can get almost everything you need out of the application store in a simple and convenient way without thinking about commands. And all of this takes a click or two to be disabled, it's not prohibiting you from using the terminal, just giving you a warning before you do something that could, or not, cause instabilities on your system (which is literally what happened to me), it simply looks like good UX design to me.

**Comment by LongJohn-Silver** [comment]
From a recent Reddit post:

> 
> We’ve recently merged some additional messaging into Onboarding that makes it clear that expected sideload method is via Flatpak apps. We don’t recommend downloading deb packages from the general web or using PPAs. These methods of software distribution are insecure and often cause stability issues. We very intentionally don’t support these methods and have no plan to support them. Flatpak is the way to go here.
> 
> You can right click on Files in the applications menu or the dock to see an “Open as admin” action, though we always recommend caution when running apps with administrator privileges.
> 
> The majority of people downloading elementary OS are coming from Windows. macOS converts are a pretty small percentage. But regardless, you can file an issue at https://GitHub.com/elementary/seeds for the inclusion of new packages.

**Comment by dawidpotocki** [comment]
elementaryOS is not a good distro (in my opinion).

A lot of applications that are not made for elementary will look broken,
because their GTK theme is total garbage. And don't try setting Adwaita,
elementary apps and Pantheon will look broken.

elementary App Store will warn you when you try to install "non-curated"
applications… which means from Ubuntu repositories, which is "curated" by
Canonical and Debian (because it's upstream). So you are supposed to only use
crappy elementary OS specific applications.

Maximize and close buttons… why are they on separate sides? It only makes you
move your mouse cursor more for absolutely no reason and gives worse UX when
some applications will just render them on one side (mostly on the right).

Epiphany web browser? It's **GNOME** Web, stop rebranding it to the old name.
Oh and it's unusable, it really is, why? It's slow, like really slow, has a lot
of rendering bugs and other bugs. For example I can't even open anything on the
localhost, it would just never load. Now, you can go and install something like
Firefox and/or Chromium. Sure, but they will look out of place. For example
Firefox's CSD (client side decoration) will render maximized and close buttons
on one side ¯\\\_(ツ)_/¯. Is this really good UX when default browser is
garbage? Every distro is just shipping Firefox or Chromium, which are both
usable browsers.

Their "Code" text editor? It's unusable for programming because it wouldn't
even auto-indent for me, so I had to manually indent every line.

Bugs, bugs. I remember trying out elementary OS on a LiveUSB, first, installer
just would crash at some step, very nice. Another bug is that it doesn't show
every application there is installed in their application menu, so to for
example open GNOME Terminal I had to type `gnome-terminal` in the Pantheon
Terminal, oh guess what happened? It was giving me notification that any
command I executed in that `gnome-terminal` finished, so it would even tell me
that `ls` finished, like no shit it took less than 1 second. It even notified
me about `Ctrl-c`. A lot of times even if I installed something, it just
wouldn't show an icon, because why would it. Another thing, wingpanel was
disappearing when clicking on the application menu and getting it back was very
hard.Their "Code" text editor? It's unusable for programming because it
wouldn't even auto-indent for me, so I had to manually indent every line.

> plus things like Firefox are not ESR (which is something horrible to use IMO)

That means you really don't like old software, guess what, elementary OS is
based on Ubuntu LTS, which is going to have a lot of outdated packages.

Use Debian Unstable (`sid`) instead of Stable (currently `buster`), it has much
newer packages and is much better for desktop.

https://wiki.debian.org/DebianUnstable/#Installation
https://packages.debian.org/sid/firefox

> ElementaryOS comes with an application store that has quite a lot more
> applications than Debian does

And all of them are elementary OS specific applications, which are most of the
time overly simplistic and you can get a better (in my opinion) application
which is made for every distro.

> That, and also their design is really welcoming and modern, in contrast with
> Trisquel which seems a bit outdated to say the least.

We don't really recommend Trisquel. It's FSF garbage.

> The default copy/paste on the terminal is ctrl+c/ctrl+v (instead of
> shift+ctrl+c/v)

And that will break stuff and introduce more confusion.

> when you run a command with sudo privileges it gives you a warning that it is
> asking for such

????

> and I assume that the ppa thing is related to that

No.

**Comment by dngray** [comment]
Closing, ElementaryOS doesn't have any features specific to a privacy audience.

We recommend people stay with more mainstream distributions with long term support/and or commercial support if necessary for desktop distributions. Those tend to also have better QA.
